src/ducks/transfers/TransferPage.jsx

```diff
@@ -109,9 +122,21 @@ const BeneficiaryRow = ({ beneficiary, onSelect }) => {
           <Text>{beneficiary.label}</Text>
           <Caption>{beneficiary.iban}</Caption>
         </Bd>
-        <Img>
-          {beneficiary.balance ? <Bold>{beneficiary.balance}</Bold> : null}
-        </Img>
+        {beneficiary.account ? (
+          <Img>
+            (
+            <Bold>
+              <Figure
+                symbol="€"
+                total={beneficiary.account.balance}
+                coloredPositive
+                coloredNegative
+                coloredWarning
+              />
+            </Bold>
+            )
```

Comment:
These parenthesis makes it harder to read for me :(

Reply:
Eagle eye ! Thx